### DISCUSSION TITLE
Code review ІМ-11

### DISCUSSION THREAD

**Discussion by yaryna-bashchak** [HowProgrammingWorks/LiveQA#43] [Code review]
Початок курсової, 1 курс.

Пишу браузерну версію гри "Змії та драбини". 
_Суть гри:_
Є пронумероване поле 10×10. Гравці по черзі кидають кубик та переміщать свою фішку. Якщо фішка зупинилась на початку драбини, то піднімається на її вершину, а якщо на голові змії - спускається на хвіст. Виграє той, хто перший дістанеться до 100.

На даний момент у [проекті](https://github.com/yaryna-bashchak/snakes-and-ladders/tree/dev) реалізовано створення поля 10×10 з поки що заданими координатами для змій та драбин на ньому і можливість гри одного гравця.

**Comment by tshemsedinov** [comment]
See code review here: https://github.com/yaryna-bashchak/snakes-and-ladders/issues/1

**Comment by yaryna-bashchak** [comment]
Буду вдячна за друге рев'ю [проекта](https://github.com/yaryna-bashchak/snakes-and-ladders/tree/dev). 
З головного додано генерацію змій і драбин на полі, можливість гри для до 10ти гравців та сповіщення про помилки.
